### ISSUE TITLE
Maven dependencies resolving to non-existent com.microsoft.rest:client-runtime:1.5.4

### ISSUE THREAD

**Issue by tobymurray** [Azure/autorest-clientruntime-for-java#447]
Not sure if this is the right place for this ticket, but we have a number of dependencies which started failing this afternoon. We're using things like [azure-mgmt-resources](https://mvnrepository.com/artifact/com.microsoft.azure/azure-mgmt-resources/1.1.2), which itself declares a dependency on [azure-client-runtime](https://mvnrepository.com/artifact/com.microsoft.azure/azure-client-runtime), with the version range 1.0.1 <= version < 2.0.0.

Up until this afternoon both Gradle and Ivy were resolving that (seemingly correctly) to the latest version or 1.5.3. They now seem to resolve it to version 1.5.4 which we mysteriously have a pom for.

I've added the `pom.xml` as it exists for me locally in a gist [here](https://gist.github.com/tobymurray/fdd2fc1cf254fdaf068aba0db2156b71) in case it has anything useful in it.

**Comment by kjlopey** [comment]
The azure-arm-client-runtime pom lines 48 shows:
```
 <dependencies>
    <dependency>
      <groupId>com.microsoft.rest</groupId>
      <artifactId>client-runtime</artifactId>
<version>1.5.4-SNAPSHOT</version>
```

This artifact version doesn't exist, thus breaking azure-arm-client-runtime's dependencies. Hopefully it gets added soon.

**Comment by chrisinmtown** [comment]
Might fix the title here, the missing artifact name is "runtime-client" not "azure-runtime-client".  Otherwise all too easily confused with azure-client-runtime which HAS a version 1.5.4 see
http://repo1.maven.org/maven2/com/microsoft/azure/azure-client-runtime/1.5.4/

**Comment by RikkiGibson** [comment]
@anuchandy did something go awry when publishing?

**Comment by tobymurray** [comment]
We worked around this in Ivy with:
```
<override org="com.microsoft.rest" module="client-runtime" rev="1.5.3" />
```
At the end of our `ivy.xml`.

**Comment by anuchandy** [comment]
@RikkiGibson @chrisinmtown, @alexstojda @kjlopey - as @tobymurray pointed this should be fixed now, client-runtime 1.5.4 is available in maven now http://search.maven.org/#artifactdetails%7Ccom.microsoft.azure%7Cazure-client-runtime%7C1.5.4%7Cjar . Let's know if you still see the issue.
